clients/src/main/java/org/apache/kafka/common/config/ConfigDef.java

```diff
@@ -68,8 +68,8 @@
  * int anotherConfig = (Integer) configs.get(&quot;config_with_validator&quot;);
  *
  * To validate the full configuration, use:
- * List&lt;Config&gt; configs = defs.validate(props);
- * The {@link Config} contains updated configuration information given the current configuration values.
+ * List&lt;ConfigValue&gt; configValues = defs.validate(props);
+ * The {@link ConfigValue} contains updated configuration information given the current configuration values.
```

Comment:
Same here. This should be commented:
`// The {@link ConfigValue} contains updated configuration information given the current configuration values.`